geode-core/src/main/java/org/apache/geode/internal/security/LegacySecurityService.java

```diff
@@ -34,19 +43,76 @@
     this.hasPeerAuthenticator = StringUtils.isNotBlank(peerAuthenticator);
   }
 
-  @Override
```

Comment:
As inherited from our (by and large) adoption of the Google Java Style Guide, we should favor including `@Override` on all methods rather than removing these three.